apps/web/app/api/mutations/useLogoutUser.ts

```diff
@@ -5,15 +5,19 @@ import { AxiosError } from "axios";
 import { useToast } from "~/components/ui/use-toast";
 import { useCurrentUserStore } from "~/modules/common/store/useCurrentUserStore";
 import { useAnnouncementsPopupStore } from "~/modules/Dashboard/store/useAnnouncementsPopupStore";
+import { useTheme } from "~/modules/Theme";
 
 import { requestManager, ApiClient } from "../api-client";
 import { queryClient } from "../queryClient";
 
 import { useAuthStore } from "./../../modules/Auth/authStore";
 
+const defaultPrimaryColor = "#3f58b6";
```

Comment:
1. I would move it to constants if possible.
2. What's the purpose of setting default once user logout?